src/components/Public/Record/Exporter/FieldPicker.vue

```diff
@@ -137,25 +137,24 @@
         {{ $t('recordList.export.recordCount', { count: getExportableCount}) }}
       </span>
       <span class="ml-auto">
-        <b-button
+        <c-input-processing
           v-if="allowJSON"
-          :disabled="exportDisabled"
-          variant="dark"
+          :processing="processing"
+          :disabled="exportDisabled || processing"
+          :label="$t('recordList.export.json')"
+          :variant="'dark'"
           class="mr-2"
           @click="doExport('json')"
-        >
-
-          {{ $t('recordList.export.json') }}
-        </b-button>
-        <b-button
+        />
+        <c-input-processing
           v-if="allowCSV"
-          :disabled="exportDisabled"
-          variant="dark"
+          :processing="processing"
+          :disabled="exportDisabled || processing"
```

Comment:
this can just be exportDisabled, since we will be disabling button in `c-input-processing` if its processing